open_tms_backend/open_tms_close_ticket/index.js

```diff
@@ -1,5 +1,176 @@
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               ticket_id:
+ *                 type: integer
+ *                 description: ID of the ticket to close
+ *                 example: 123
+ *     responses:
+ *       200:
+ *         description: Ticket closed successfully
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 meta:
+ *                   type: object
+ *                   properties:
+ *                     status:
+ *                       type: integer
+ *                       example: 200
+ *                     message:
+ *                       type: string
+ *                       example: Ticket closed successfully
+ *                 activity:
+ *                   $ref: '#/components/schemas/TicketActivity'
+ *       400:
+ *         description: Bad Request
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 meta:
+ *                   type: object
+ *                   properties:
+ *                     status:
+ *                       type: integer
+ *                       example: 400
+ *                     message:
+ *                       type: string
+ *                       example: Bad Request
+ *                 errors:
+ *                   type: array
+ *                   items:
+ *                     type: object
+ *                     properties:
+ *                       code:
+ *                         type: string
+ *                         example: BAD_REQUEST
+ *                       message:
+ *                         type: string
+ *                         example: Next stage or assignee is invalid.
+ *       500:
+ *         description: Internal Server Error
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 meta:
+ *                   type: object
+ *                   properties:
+ *                     status:
+ *                       type: integer
+ *                       example: 500
+ *                     message:
+ *                       type: string
+ *                       example: Something went wrong.
+ *                 errors:
+ *                   type: array
+ *                   items:
+ *                     type: object
+ *                     properties:
+ *                       code:
+ *                         type: string
+ *                         example: INTERNAL_ERROR
+ *                       message:
+ *                         type: string
+ *                         example: Something went wrong.
+ */
+const closeTicket = async (ticketId) => {
+  const TicketRevision = await shared.import('open_tms', 'TicketRevision')
```

Comment:
You can get the shared prisma client and not the db models directly from shared.Use a dynamic import to get shared blocks
 ```js
   const { prisma } = await shared.getShared();
  ```